octobot_trading/exchanges/types/websocket_exchange.py

```diff
@@ -143,9 +143,9 @@ async def start_sockets(self):
             except ValueError as e:
                 self.logger.error(f"Failed to start websocket on {self.exchange_name} : {e}")
 
-        if not self.is_websocket_running:
+        if self.websocket_connectors and not self.is_websocket_running:
             self.logger.error(f"{self.exchange_manager.exchange_name.title()}'s "
```

Comment:
Are we sure we want an error here (when nothing to watch) ?

Reply:
right, we switch it to debug ?

Reply:
Yes :+1: